docs/wiki/FAQ.md

```diff
@@ -152,3 +153,19 @@ The Management Group Names/IDs created via the ALZ Portal Accelerator Deployment
     - `<Resource Prefix (Root ID)>-corp`
   - `<Resource Prefix (Root ID)>-decommissioned`
   - `<Resource Prefix (Root ID)>-sandbox`
+
+## Why hasn't Azure landing zones migrated to the Azure Monitor Agent yet?
+
+Great question! Don't worry we are aware of this required migration and change to Azure landing zones with the Log Analytics Agent (Microsoft Monitoring Agent - MMA) being retired in August 2024 as detailed here: [Migrate to Azure Monitor Agent from Log Analytics agent](https://learn.microsoft.com/azure/azure-monitor/agents/azure-monitor-agent-migration).
+
+We are working hard internally with the Azure Monitor Product Group (PG) to ensure everything that Azure landing zones requires and gets from the Log Analytics Agent (Microsoft Monitoring Agent - MMA) approach today is covered and has a path for migration to the Azure Monitor Agent (AMA) approach. This has been underway for sometime and continues to progress.
+
+The AMA agent bring a number of number concepts, resources and changes to existing integrations with other services, such as Microsoft Defender for Cloud, that all require validation by each of the associated PGs as well as the Azure landing zone team, prior to migrating to AMA from MMA.
```

Comment:
Can't add a suggestion for some reason but change 'number' to 'new'

Reply:
done